src/redux/Delivery/actions.js

```diff
@@ -15,6 +15,7 @@ export const LOAD_TIME_SLOT_SUCCESS = '@delivery/LOAD_TIME_SLOT_SUCCESS';
 export const LOAD_TIME_SLOTS_SUCCESS = '@delivery/LOAD_TIME_SLOTS_SUCCESS';
 export const SET_REFRESHING = '@delivery/SET_REFRESHING';
 export const SET_STORE = '@delivery/SET_STORE';
+export const SET_STORES = '@delivery/SET_STORES';
```

Comment:
Not a blocker, but I think we should get rid of these extra variables and define actions as:
`export const setStores = createAction('@delivery/SET_STORES');`

(When you need an action name inside a reducer you can use `case setStores.type:`)